scripts/ccvi/parse_ccvi_counties.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * A one-time script to parse Surgo foundation's CCVI data for counties and states (ccvi_counties.csv, ccvi_states.csv)
+ * https://precisionforcovid.org/ccvi
+ * https://surgoventures.org/resource-library/report-vulnerable-communities-and-covid-19
+ *
+ * Results in ccvi-data.json with a format of [fips]: {CCVI data}
+ */
+import fs from 'fs';
+import path from 'path';
+import { toNumber } from 'lodash';
+import { FipsToCcviMap } from '../../src/common/data';
+const promisify = require('util').promisify;
+const csvParse = promisify(require('csv-parse'));
+
+const outputPath = path.join(__dirname, '../../src/common/data/ccvi-data.json');
+
+function formatFips(fips: string): string {
+  if (fips.length === 4 || fips.length === 1) {
+    return `0${fips}`;
+  }
+  return fips;
+}
+
+async function main() {
+  const fipsToCcviData: FipsToCcviMap = {};
+
+  const statecCsvText = fs.readFileSync(
+    path.join(__dirname, 'ccvi_states.csv'),
+  );
+  const [, ...stateRows] = await csvParse(statecCsvText);
+
+  const countyCsvText = fs.readFileSync(
+    path.join(__dirname, 'ccvi_counties.csv'),
+  );
+  const [, ...countyRows] = await csvParse(countyCsvText);
```

Comment:
nit: Maybe create a function to avoid repetition here? Also, there is a typo in `statecCsvText` → `stateCsvText`

```ts
const stateRows = readCsv(stateCsvPath);
const countyRows = readCsv(countyCsvPath);
```